skimage/data/__init__.py

```diff
@@ -7,45 +7,50 @@
 """
 
 import os as _os
+import warnings
 
 from .. import data_dir
 from ..io import imread, use_plugin
 from .._shared.utils import deprecated
 from ._binary_blobs import binary_blobs
+from .. import img_as_bool
 
 __all__ = ['load',
+           'astronaut',
            'camera',
-           'lena',
-           'text',
            'checkerboard',
-           'coins',
-           'moon',
-           'page',
-           'horse',
-           'clock',
-           'immunohistochemistry',
            'chelsea',
+           'clock',
            'coffee',
+           'coins',
+           'horse',
            'hubble_deep_field',
-           'rocket',
-           'astronaut']
+           'immunohistochemistry',
+           'lena',
```

Comment:
I thought we kept the `lena` function name - just the name! - for API backwards compatibility. That way we can throw a useful warning when people try to use it, telling them why it's gone (non-free) and directing them at `astronaut` which is both free and a much superior test image to boot.

Always better to tell users what's going on, IMHO.